packages/components/src/components/dataDisplay/NumberedList/NumberedList.tsx

```diff
@@ -33,7 +33,7 @@ const NumberedListItem: FC<NumberedListItemProps> = ({
     <Box>
       {Children.map(children, (child) => {
         if (isValidElement(child) && child.type === NumberedList) {
-          return cloneElement(child, { number, paddingTop: '0.75rem' });
+            return cloneElement(child, { number, paddingTop: '0.75rem' } as NumberedListProps);
```

Comment:
Is this typecast necessary?, also there is an extra indentation that does not follow the rest of the repo